### PR TITLE
Jetty update 4.23

### PR DESCRIPTION
**Pull request by gireeshpunathil**

### CODE REVIEW

File: org.eclipse.help.webapp/pom.xml [hunk 1]
```diff
@@ -25,7 +25,7 @@
       <plugin>
         <groupId>org.eclipse.jetty</groupId>
         <artifactId>jetty-jspc-maven-plugin</artifactId>
-        <version>10.0.6</version>
+        <version>10.0.14</version>
```
Comment (sravanlakkimsetti) [inline org.eclipse.help.webapp/pom.xml:28]: shouldn't this be 10.0.15?

**Review by sravanlakkimsetti — CHANGES_REQUESTED** [review]
shouldn't this be 10.0.15?

Also you'll need to increment the bundle version

**Comment by sravanlakkimsetti** [comment]
another issue please target this patch to 4.23_maintenance branch

File: org.eclipse.help.webapp/pom.xml (hunk 1 above)
Comment (gireeshpunathil) [reply org.eclipse.help.webapp/pom.xml:28]: https://github.com/eclipse/jetty.project/security/advisories/GHSA-p26g-97m4-6q7c

I was following this document, which says: `<=9.4.50, <=10.013 : 9.4.51, 10.0.14, 11.0.14`

Should I move up to `10.0.15`?

**Pull request closed without merging**